skins/laika/src/logger.ts

```diff
@@ -0,0 +1,49 @@
+import { Notifier } from '@airbrake/browser';
+
+const LOGGER_ERRBIT: string = 'errbit';
+const LOGGER_CONSOLE: string = 'console';
+
+interface Logger {
+	notify( error: object ): void
+}
+
+class ErrbitLogger implements Logger {
+	notifier: Notifier;
+
+	constructor( host: string, projectId: number, projectKey: string ) {
+		this.notifier = new Notifier( {
+			host: host,
+			projectId: projectId,
+			projectKey: projectKey,
+		} );
+	}
+
+	notify( error: object ) {
+		this.notifier.notify( error );
+	}
+}
+
+class SilentLogger implements Logger {
+	notify( error: object ) {}
+}
+
+class ConsoleLogger implements Logger {
+	notify( error: object ) {
+		console.log( error );
+	}
+}
+
+export default function createLogger(): Logger {
+	switch ( process.env.VUE_APP_LOGGER ) {
+		case LOGGER_CONSOLE:
+			return new ConsoleLogger();
+		case LOGGER_ERRBIT:
+			return new ErrbitLogger(
+				process.env.VUE_APP_ERRBIT_HOST || '',
+				parseInt( process.env.VUE_APP_PROJECT_ID || '' ),
```

Comment:
When I tested errbit with the live setup it broke and I discovered that the [docs were wrong](https://github.com/errbit/errbit/issues/1452). So that parameter is now hardcoded.